script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -472,7 +477,8 @@ It prevents me from committing to the urge to kill him,
 [sha:93549664554ac3174a998535b6d89d54dbc22eca]{
 -- Page 60, Offset 30687.
 -- 人生はあまりにも曖昧だ。
-Life is such a curious thing.
+// group pr suggestion: How enigmatic life is.
+Life is an enigma.
```

Comment:
Life is such an enigma

Reply:
tbqh I prefer the original line over all 3 of these suggestions but of the three, I'd pick Yusuke's